pycbc/tmpltbank/partitioned_bank.py

```diff
@@ -583,6 +583,28 @@ def add_tmpltbank_from_xml_table(self, sngl_table, vary_fupper=False):
             self.add_point_by_masses(sngl.mass1, sngl.mass2, sngl.spin1z,
                                      sngl.spin2z, vary_fupper=vary_fupper)
 
+    def add_tmpltbank_from_hdf_file(self, hdf_fp, vary_fupper=False):
+        """
+        This function will take a pointer to an open HDF File object containing
+        a list of templates and add them into the partitioned template bank
+        object.
+
+        Parameters
+        -----------
+        hdf_fp : h5py.File object
+            The template bank in HDF5 format.
+        vary_fupper : False
+            If given also include the additional information needed to compute
+            distances with a varying upper frequency cutoff.
+        """
+        mass1s = hdf_fp['mass1'][:]
+        mass2s = hdf_fp['mass2'][:]
+        spin1zs = hdf_fp['spin1z'][:]
+        spin2zs = hdf_fp['spin2z'][:]
+        for idx in xrange(len(mass1s)):
+            self.add_point_by_masses(mass1s[idx], mass2s[idx], spin1zs[idx],
+                                     spin2zs[idx], vary_fupper=vary_fupper)
+
```

Comment:
My response got a bit long, so I summarize: I think the question is "Can this function read in output from sbank with the latest changes?". To which the answer is "This function shouldn't read in output from sbank at all, even before latest changes."

No. Varying lower-frequency cutoff breaks a whole bunch of things that are needed for PyCBC's metric approximation, so that doesn't make sense here. This function should *not* be used to read in any banks made outside of the 3 pycbc bank generation codes. It still works for what it is designed for.

.... One could consider adding a variable f_lower (as we vary f_upper) for the aligned_stochastic_bank pycbc code. This isn't something I'm planning on implementing myself while aligned_stoch_bank has not been used in production analyses. But if this functionality is added, adding it also here would be part of the patch.